snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/literal1.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+public class Example
+{
+   public static void Main()
+   {
+      int n = 9;
+      Console.WriteLine($@"{n:##.0\%}");  
```

Comment:
Maybe it's a nit: `n` is of integer type, while format string assumes fractional digits. Also it might give impression that ".0" part is constant, while it's part of the format.

What if we use `double`
```csharp
double n = 9.3;
Console.WriteLine($@"{n:##.0\%}");
```

which outputs "9.3%"?